src/database/utilities/createRecord.js

```diff
@@ -780,8 +782,55 @@ const createRequisitionItem = (database, requisition, item, dailyUsage, stockOnH
     sortIndex: requisition.items.length + 1,
   });
 
+  if (program && type === 'response') {
```

Comment:
I think a couple code comments explaining the intent here would be good. Like `// Need to save the stockOnHand of last requisitionItem of last response to the openingStock of new requisitionItem` sort of thing. I'm not sold on my phasing here haha.

Reply:
f39f5aaa9ee0e11b203c5d5d1c18dd97487a138d me either